react/components/ErrorPage/messages/bg.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
shouldn't we need those translations before merging the PR?

Reply:
For now it just falls back to english so nothings is changed